UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs

```diff
@@ -4,17 +4,38 @@
 using GLTF.Schema;
 using UnityEngine;
 using UnityGLTF.Extensions;
+using UnityEditor;
 
 namespace UnityGLTF
 {
 	public class GLTFSceneExporter
 	{
+        enum TextureMapType
```

Comment:
Why not add a Boolean that indicates whether the image is sRGB or not instead of a type and then a switch to determine it?

Reply:
This was part of @justinctlam's changes. I just didn't want to add more to the code, but I can definitely do that. Thanks for pointing that out!

Reply:
I don't follow how my changes are related to sRGB?

Reply:
They do not, which is why I added it. We need to export textures in the proper color space, or they won't look right.